src/components/Sort/Sort.tsx

```diff
@@ -39,46 +35,34 @@ export interface ISortProps {
  *
  * Expects to be controlled (i.e. using sort and onChange to control value/updating)
  */
-export const Sort = (props: ISortProps): ReactElement => {
+export const Sort = <S extends SortType, F extends SortField>(props: ISortProps<S, F>): ReactElement => {
```

Comment:
```suggestion
export const Sort = <S extends SortType = SolrSort, F extends SortField = SolrSortField>(props: ISortProps<S, F>): ReactElement => {
```

We could make solrSort the default so you don't have to pass the generic everywhere